tool/site/server_hapi/src/services/session.ts

```diff
@@ -25,7 +25,19 @@ export function setSessionValue(key: string, value: unknown, request: Hapi.Reque
     value = {arrayValues: value}
   }
   if (isLocal()) {
-    console.error(`Saving ${key} to session with value: ${JSON.stringify(value)}`)
+    console.log(`Saving ${key} to session with value: ${JSON.stringify(value)}`)
   }
   request.yar.set(key, value)
 }
+
+export function appendToSessionValue(key: string, value: unknown, request: Hapi.Request): void {
+  const existingValue = getSessionValueOrDefault(key, request, [])
+  if (!Array.isArray(existingValue)) {
+    throw Error("Cannot append to session value, session value is not an array")
+  }
+  const mergedValue = existingValue.concat(value)
+  setSessionValue(key, mergedValue, request)
+  if (isLocal()) {
+    console.log(`Saving ${key} to session with value: ${JSON.stringify(mergedValue)}`)
```

Comment:
think this line will be logged out twice, once here and once when calling `setSessionValue`

Reply:
good spot, removed in next pr